pydantic/errors.py

```diff
@@ -216,7 +218,9 @@ class UrlExtraError(UrlError):
     msg_template = 'URL invalid, extra characters found after valid URL: {extra!r}'
 
 
-class EnumError(PydanticTypeError):
+class EnumMemberError(PydanticTypeError):
+    code = 'enum'
```

Comment:
I kept the code `enum` to avoid breaking the tests of *fastAPI*.
I first named it `enum_member` to use `enum` and `int_enum` for added errors but I'm not sure it's worth a change. WDYT?